exampleCourse/questions/examplesAddCodeFromFile/question.html

```diff
@@ -0,0 +1,19 @@
+
+<pl-question-panel>
+  <p>
+    Consider the following code snippet:
+ </p>
+ <pl-code language="python" input-file-name="myCode.py">
+</pl-code>
```

Comment:
Nit: if an element has no content, I prefer to put its closing tag on the same line as the opening tag.